data/MI/incentives.json

```diff
@@ -0,0 +1,1400 @@
+      "number": 750
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$750 for 18 SEER or 17-17.99 SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-23",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$250 for 16-17 SEER or 15.2-16.99 SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-24",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 150
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$150 for 15 SEER or 14.3-15.19 SEER2 Air Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-25",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 800
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$800 for 17+ SEER or 16+ EER2 Ground Source Heat Pump."
+    }
+  },
+  {
+    "id": "MI-26",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "mi_dTE_airConditioners",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1000 for 18+ SEER and HSPF 8.8+  OR  17+ SEER2 and HSPF2 7.5+."
+    }
+  },
+  {
+    "id": "MI-27",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "mi_dTE_wi-FiEnabledThermostats",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$50 for new Wi-Fi enabled thermostat."
+    }
+  },
+  {
+    "id": "MI-28",
+    "authority_type": "utility",
+    "authority": "mi-dte",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "mi_dTE_insulation&Windows",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 125
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$125 for Roof & Attic insulation. Must insulate a minimum of 500 square feet of attic area."
```

Comment:
More excessive capitalization on some words, here and in the next several. Also some of them need the word "insulation" added -- e.g. "above-grade wall insulation".